### DISCUSSION TITLE
Docs is dead sometimes

### DISCUSSION THREAD

**Discussion by catchellers** [vercel/next.js#55192] [Help]
### Summary

I tried to access https://nextjs.org/ but sometimes this website had "ERR_ADDRESS_UNREACHABLE", i dont know the reason why my IP or difference things

**Comment by duoluodexiaoxiaoyuan** [comment]
yeah, I occur happen, i don't know how why?

**Comment by Uy575** [comment]
go to Settings > System on your device. Next, navigate to Troubleshoot > Other troubleshooters and scroll down to Windows Store Apps 
run troubleshoot then try again hope it will work.

**Comment by duoluodexiaoxiaoyuan** [comment]
> go to Settings > System on your device. Next, navigate to Troubleshoot > Other troubleshooters and scroll down to Windows Store Apps转到设备上的设置>系统。接下来，导航到疑难解答>其他疑难解答，然后向下滚动到Windows应用商店应用 run troubleshoot then try again hope it will work. 运行故障排除，然后再试一次，希望它会工作。

thank u replay
